  fun ref return_error(error': GError): None =>
    GTaskP.return_error(this, error')
